Week7/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/App.kt

```diff
@@ -2,18 +2,30 @@ package com.raywenderlich.myfavoritemovies
 
 import android.app.Application
 import androidx.room.Room
+import com.raywenderlich.myfavoritemovies.data.networking.RemoteApi
+import com.raywenderlich.myfavoritemovies.data.networking.buildApiService
 import com.raywenderlich.myfavoritemovies.model.DATABASE_NAME
 import com.raywenderlich.myfavoritemovies.model.MovieDatabase
 
 /**
  * Using an App class, so that I can init the movieDb one time.
  */
+
+private const val TOKEN = "08e2d505-51b6-4bae-8fc3-a12a0cbe8951"
```

Comment:
For best approaches, you want to have these keys in a `local.properties` file or another secure file, and load it in Build time! That way you don't push your API keys to public repos!